builds/e2e/longhaul.yaml

```diff
@@ -14,6 +14,8 @@ variables:
   #container.registry: ''
   #snitch.storage.account: ''
   # Variables settable at queue time
+  #test.startDelay.arm
+  #test.startDelay.amd
```

Comment:
Should we model startDelay to 2 value- 1 for 32 bits and 1 for 64 bits? or model 1 for short (test.startDelay.short) and 1 for long (test.startDelay.long)?  If you model use architecture, we will need to add arm64 in the future.

Reply:
I think that it is fine unless there exist amd platforms that we want a long timeout for (or vice versa).